src/stories/radioButton/notSelected.stories.js

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import RadioButton from "../../components/radioButton"
+import { action } from "@storybook/addon-actions"
+
+const onChange = () => action("onChange")
+const initialValue = "B"
+
+export default {
+  title: "Radio Button/Not selected",
+  component: RadioButton,
+  args: {
+    storyName: "",
+    value: initialValue,
+    checked: false,
```

Comment:
we discussed this in another PR, but maybe I missed the decision. I really don't see the value of adding all variations of checkboxes with checked/unchecked. a knob for checking each would be enough for me

Reply:
Or have one checked representation of the default one. If I remember correctly, the checked checkboxes have all the same style in each variation

Reply:
We had this discussion for the Checkbox and decided to not show each variation. 
I thought of doing the same for the RadioButton but I doubted as it behaves differently when checked or unchecked : the click on an unchecked radioButton will trigger an action but the click on a checked one won't trigger anything.
Actions might not be that important in Storybook though. What do you think ?

Reply:
Yeah, I would do the same for radiobutton and not do checked examples